starters/nuxt3-pinia-vuetify/components/FetchExample/FetchExample.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="fetch-example-box d-flex flex-column justify-center align-center mt-5">
+    <div class="fetch-example-box__header border-b-4 border-blue-600 mb-4 text-center">
+      <h4 class="text-h5 font-weight-bold">
+        Fetch Data From API
+      </h4>
+    </div>
+
+    <div class="d-flex flex-row justify-center align-center mt-5 mb-10 text-h6">
+      <p>Message:</p>
+      <p class="ml-5 font-weight-bold" data-testid="message-value">
+        {{ message }}
```

Comment:
We should add a skeleton placeholder while the message is loading here!

Reply:
updated